webapp/channels/src/components/integrations/abstract_outgoing_webhook.test.tsx

```diff
@@ -0,0 +1,90 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import {shallow} from 'enzyme';
+
+import AbstractOutgoingWebhook from 'components/integrations/abstract_outgoing_webhook';
+import {Team} from '@mattermost/types/teams';
+
+describe('components/integrations/AbstractOutgoingWebhook', () => {
+    const team: Team = {
```

Comment:
**Note:** Apologies for it breaking this out into a new file. None of the test test functionality has changed I just fleshed out the props a bit and modeled them after the tests in `abstract_incoming_webhook.test.tsx`: https://github.com/mattermost/mattermost/blob/96d3e5864f95065e4845101ad3cabc9cb9a58f35/webapp/channels/src/components/integrations/abstract_incoming_hook.test.tsx#L12

I was considering adding additional tests here as they're a bit sparse, but didn't want to bloat the size of this PR. If there's interest in me adding some additional unit tests for this component just let me know, can be here or in another PR. 🙂

Reply:
it would have been good with git diff if the file was just renamed instead of deleting and adding. But its fine :)